starter-code/basic-algorithms.js

```diff
@@ -1,3 +1,45 @@
+var hacker1 = prompt("Enter the driver's name"); console.log("The driver's name is " + hacker1); 
+
+
+var hacker2 = prompt("Enter the navigator's name"); console.log("The navigator's name is " + hacker2); 
+  if (hacker1.length === hacker2.length) {
+    console.log("wow, you both got equally long names, " + hacker1.length + " characters!!") } 
+  else if (hacker1.length <= hacker2.length) { 
```

Comment:
here you don't want to be checking if it's greater than or equal to- you only want to check to see if it is greater than since you're checking for the longest name